dali/operators/generic/reduce/reduce.cc

```diff
@@ -30,14 +34,53 @@ Not providing any axis results in reduction of all elements.)code",
     "If True, maintains original input dimensions.",
     false);
 
-DALI_SCHEMA(reductions__Sum)
-  .DocStr("Gets sum of elements along provided axes.")
+DALI_SCHEMA(ReduceWithOutputType)
   .AddOptionalArg("dtype",
     R"code(Output data type. This type is used to accumulate the result.)code",
     DALI_NO_TYPE)
+  .AddParent("ReduceBase");
+
+  DALI_SCHEMA(ReduceWithMeanInput)
+  .AddOptionalArg("ddof",
+    R"code(Delat Degrees of Freedom. Adjusts the divisor used in calculations, which is `N - ddof`.)code",
+    0)
+  .AddParent("ReduceWithOutputType");
+
+DALI_SCHEMA(reductions__Std)
```

Comment:
I don't like this capitalization - I think it should be StD, but it would require some special handling in _to_snake_case (it's too short to add to special case mapping with acceptable risk).

Reply:
Done, changed to `STD`